src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1007,22 +1007,30 @@ internal NamedTypeSymbol GetSpecialType(SpecialType typeId, DiagnosticBag diagno
         /// </summary>
         internal Symbol GetSpecialTypeMember(SpecialMember member, DiagnosticBag diagnostics, CSharpSyntaxNode syntax)
         {
-            var memberSymbol = Compilation.GetSpecialTypeMember(member);
-            if ((object)memberSymbol != null)
+            Symbol memberSymbol;
+            return TryGetSpecialTypeMember(this.Compilation, member, syntax, diagnostics, out memberSymbol)
+                ? memberSymbol
```

Comment:
Well, the TryGet method places null into its out parameter when it returns false, so you can simplify this to

```cs
TryGetSpecialTypeMember(this.Compilation, member, syntax, diagnostics, out memberSymbol);
return memberSymbol;
```

Though I admit that may be less clear.